Update troubleshooting.md (#38022)
DataDog/documentation: content/en/database_monitoring/setup_sql_server/troubleshooting.md

sqlcmd -S <INSTANCE_ENDPOINT> -d master -E
```

> If the `datadog` user is unable to log into the SQL Server instance, please ensure the user has been created and given the proper permissions according the [setup documentation][1]. Windows Authentication is supported when the Agent connects to a SQL Server instance running on the same host as the Agent.

Reviewer: ```suggestion
If the `datadog` user is unable to log in to the SQL Server instance, verify the user has been created and given the proper permissions according the [setup documentation][1]. Windows Authentication is supported when the Agent connects to a SQL Server instance running on the same host as the Agent.
```
